packages/frontend-shared/cypress/support/onlineNetwork.ts

```diff
@@ -0,0 +1,62 @@
+export function goOnline () {
+  Cypress.Commands.add('goOnline', () => {
+    cy.log('**go online**')
+    .then(() => {
+      Cypress.automation('remote:debugger:protocol',
+        {
+          command: 'Network.emulateNetworkConditions',
+          params: {
+            offline: false,
+            latency: -1,
+            downloadThroughput: -1,
+            uploadThroughput: -1,
+          },
+        })
+    })
+    .then(() => {
+      Cypress.automation('remote:debugger:protocol',
+        {
+          command: 'Network.disable',
+        })
+    })
+  })
+}
+
+export function goOffline () {
+  Cypress.Commands.add('goOffline', () => {
+    cy.log('**go offline**')
+    .then(() => {
+      return Cypress.automation('remote:debugger:protocol',
+        {
+          command: 'Network.enable',
+        })
+    })
+    .then(() => {
+      Cypress.automation('remote:debugger:protocol',
+        {
+          command: 'Network.emulateNetworkConditions',
+          params: {
+            offline: true,
+            latency: -1,
+            downloadThroughput: -1,
+            uploadThroughput: -1,
+          },
+        })
+    })
+  })
+}
+
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      /**
+      * Simulates offline network mode
+      */
+        goOffline(): void
```

Comment:
Generally you want to return `Chinable` from a custom command, then we can continue doing

```ts
cy.get()
.goOffline()
.goOnline()
```

etc. Example here: https://github.com/cypress-io/cypress/blob/edb4642328d6a6085df12dc246df8cfa3841cc5f/packages/reporter/index.d.ts#L16

In this case it might just be `Chainable<void>`